helm/hpcc/templates/eclagent.yaml

```diff
@@ -79,7 +86,7 @@ data:
 {{- $apptype := .type | default "hthor" -}}
 {{- $secretsCategories := list "system" "ecl-user" "ecl" "storage" }}
 {{- $commonCtx := dict "root" $ "me" . "secretsCategories" $secretsCategories }}
-{{- $configSHA := include "hpcc.getConfigSHA" ($commonCtx | merge (dict "configMapHelper" "hpcc.agentConfigMap" "component" "eclagent" "excludeKeys" (print "global," $apptype ".replicas"))) }}
+{{- $configSHA := include "hpcc.getConfigSHA" ($commonCtx | merge (dict "configMapHelper" "hpcc.agentConfigMap" "component" "eclagent" "excludeKeys" (print "global," $apptype ".replicas" ".env"))) }}
```

Comment:
I may be missing something.. if .env changes, pod will restart anyway, so valid to ignore, but also no point?

Reply:
I wanted to suppress it from the config file because confusing to have it there when it isn't read from there.

Reply:
ok, it will still be in the configmap with this change, just not part of the calculated SHA.
Would need an alternative form of generateConfig (based on getConfigSHA) that generated the filtered output, instead of piping into sha256sum for that.